Packs/Base/Scripts/ValidateContent/README.md

```diff
@@ -59,7 +59,7 @@ This automation script is used as part of the content validation that runs as pa
 
 ### Validation Results
 
-|Name|Error|Line|
-|---|---|---|
-| MyScript | The docker image tag is not the latest numeric tag, please update it. | |
-| MyScript | test for membership should be 'not in' | 44 |
+| Name     | Error                                                                                                                                                                                         | Line | 	Error Code/Linter |
+|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------|--------------------|
+| MyScript | The following commands contain duplicated arguments: Command gpt-send-message, contains multiple appearances of the following arguments message. Please make sure to remove the duplications. |      | IN113              |
```

Comment:
`gpt-send-message`  is relevant or an example command? if example use a general example command:
Command example-my-command, contains...